fastplotlib/graphics/utils.py

```diff
@@ -0,0 +1,21 @@
+from contextlib import contextmanager
+
+from ._base import Graphic
+
+
+@contextmanager
+def pause_events(*graphics: Graphic):
+    if not all([isinstance(g, Graphic) for g in graphics]):
+        raise TypeError(
+            f"`pause_events` only takes Graphic instances as arguments, "
+            f"you have passed the following types:\n{[type(g) for g in graphics]}"
```

Comment:
Should we make this a set so we are only returning the different unique types passed? 

I can just think of maybe wanting to pass a lot of graphics to be paused at one time and then finding it hard to figure out which type might be erroneous

Reply:
If it's like this then the list of pass types is printed in the same order as the user's argument so they can figure out which item is wrong. Whereas if they passed a set you don't know which index has the non-Graphic item.